pulp_rpm/app/viewsets/repository.py

```diff
@@ -225,10 +225,13 @@ def sync(self, request, pk):
             )
             if repository.retain_package_versions > 0:
                 raise DRFValidationError(err_msg.format("retain_package_versions"))
-        elif sync_policy == SYNC_POLICIES.MIRROR_COMPLETE:
+
+        if sync_policy == SYNC_POLICIES.MIRROR_COMPLETE:
```

Comment:
I think this was a separate bug?  The cases here were never being tested because they fell through the first branch

Reply:
nice catch